source/appModules/soffice.py

```diff
@@ -318,6 +318,43 @@ def _get_locationText(self):
 class SymphonyDocument(CompoundDocument):
 	TextInfo = SymphonyDocumentTextInfo
 
+	# override base class implementation because that one assumes
+	# that the text retrieved from the text info for the text unit
+	# is the same as the text that actually gets removed, which at
+	# least isn't true for Writer paragraphs when removing a word
+	# followed by whitespace using Ctrl+Backspace
+	def _backspaceScriptHelper(self, unit, gesture):

```

Comment:
could this function please get type hints added?

```suggestion
	def _backspaceScriptHelper(self, unit: str, gesture: inputCore.InputGesture):
```

Reply:
Done in the new version.